Source/Core/Core/TitleDatabase.cpp

```diff
@@ -91,8 +94,26 @@ TitleDatabase::TitleDatabase()
 
 TitleDatabase::~TitleDatabase() = default;
 
+const std::string& TitleDatabase::GetDeveloper(const std::string& gametdb_id) const
+
+{
+  auto it = m_user_developer_map.find(gametdb_id);
+  if (it != m_user_developer_map.end())
+    return it->second;
+
+  if (!SConfig::GetInstance().m_use_builtin_title_database)
+    return EMPTY_STRING;
+
+  it = m_base_map.find(gametdb_id);
```

Comment:
Isn't this the map that contains game names, meaning that a game name could show up in the Developer column if you don't have the game in developers.txt?

Reply:
Thank you for the clarification, I will re-work this to the appropriate category of publisher and my oversights for using custom metadata or not.